This code provokes the compiler to crash with NPE:

public class Test {
    void m() {
        try {}
        catch (Exception e) {}
    }
}